api/loginApiClient.module.ts

```diff
@@ -0,0 +1,27 @@
+﻿const API_URL = "https://testswitch-api-staging.herokuapp.com/candidates"; //TODO change to /signin when endpoint exists
+
+async function loginCredentialsIncorrect(): Promise<boolean>{
+    const response = await fetch(API_URL);
+    return response.status == 200;
+
+    //TODO add if(response.status == 403) then not authorized, anything else is another issue.
+}
+
+export async function apiTryLogin(userId: string, password: string): Promise<boolean> {
```

Comment:
We talked about potentially renaming this function. Perhaps it should be on the login form instead of the API client as well, since it is doing UI actions.